add_new_car.py

```diff
@@ -23,11 +23,11 @@ def show_add_new_car_form(id_race):
 			result = db_functions.car_exists(user.db_id, id_race)
 			if result:
 				flash('Na tento závod už auto nabízíš;)', "error")
-				return render_template('newautoinput.html', zavody=races, id_vybraneho=int(id_race), values={})
+				return render_template('add_new_car.html', zavody=races, id_vybraneho=int(id_race), values={})
```

Comment:
Since you are translating variables in the templates as well, you also must rename them in `render_template()`, since the name here (`zavody`, `id_vybraneho`, etc.) actually determines the name of the variable in the template.

This applies to all the other calls to `render_template()`, not just this one...